bindings/c/test/apitester/TesterApiWorkload.cpp

```diff
@@ -20,6 +20,7 @@
 
 #include "TesterApiWorkload.h"
 #include "TesterUtil.h"
+#include "test/fdb_api.hpp"
```

Comment:
I suppose it can be `<fdb_api.hpp>` now.

Reply:
This is a question of style, and we should have a FDB-wide policy. From my point of view, the distinction between `#include "foo"` and `#include <foo>` should suggest to the reader where to go look for the header file in question. I think a distinction that makes sense is that `<foo>` should be used for system-level headers (i.e., everything outside of your repository), and `#include "foo"` for files inside the repository.  

I would moreover like to propose (in a separate PR) to use paths relative to the repo root, as that makes it immediately obvious where to find the header (for example, `#include "bindings/c/test/fdb_api.hpp"`), as is already used in several places (e.g., https://github.com/apple/foundationdb/blob/main/bindings/c/test/apitester/fdb_c_api_tester.cpp#L28 )

As it stands, it's very confusing which header we're including (especially since we have conflicting file names, like `./bindings/c/test/unit/fdb_api.hpp` and `./bindings/c/test/fdb_api.hpp`).

Reply:
Generally, I like be able to understand code without having to rely on tooling, or having to look up extra information. Full header path unambiguously specifies which header is being used, so this is preferred for me here.

If the C++ wrapper was an official FDB library, and we were writing example code for FDB clients to follow, that would be a different story.